<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SYS0081" name="Login-test" tests="3" failures="1" timestamp="15 Sep 2014 09:06:07 GMT" time="85.288" errors="0">
  <testcase name="checkUserLogin" time="11.067" classname="com.diw.test.LoginTest"/>
  <testcase name="checkUserLogout" time="3.603" classname="com.diw.test.LoginTest"/>
  <testcase name="checkRegistration" time="60.287" classname="com.diw.test.RegistrationTest">
    <failure type="java.lang.AssertionError" message="Time out exception is occurred">
      <![CDATA[java.lang.AssertionError: Time out exception is occurred
	at com.diw.action.UserActions.getWebElement(UserActions.java:55)
	at com.diw.action.UserActions.getText(UserActions.java:106)
	at com.diw.test.RegistrationTest.checkRegistration(RegistrationTest.java:35)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- checkRegistration -->
</testsuite> <!-- Login-test -->
